br/pkg/task/operator/cmd.go

```diff
@@ -0,0 +1,101 @@
+package operator
+
+import (
+	"context"
+	"crypto/tls"
+	"strings"
+	"time"
+
+	"github.com/pingcap/log"
+	"github.com/pingcap/tidb/br/pkg/logutil"
+	"github.com/pingcap/tidb/br/pkg/pdutil"
+	"github.com/pingcap/tidb/br/pkg/task"
+	"github.com/pingcap/tidb/br/pkg/utils"
+	"go.uber.org/zap"
+	"golang.org/x/sync/errgroup"
+)
+
+func dialPD(ctx context.Context, cfg *task.Config) (*pdutil.PdController, error) {
+	pdAddrs := strings.Join(cfg.PD, ",")
+	var tc *tls.Config
+	if cfg.TLS.IsEnabled() {
+		var err error
+		tc, err = cfg.TLS.ToTLSConfig()
+		if err != nil {
+			return nil, err
+		}
+	}
+	mgr, err := pdutil.NewPdController(ctx, pdAddrs, tc, cfg.TLS.ToPDSecurityOption())
+	if err != nil {
+		return nil, err
+	}
+	return mgr, nil
+}
+
+func cleanUpWith(f func(ctx context.Context)) {
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
```

Comment:
Does 30 seconds enough for operator to start all jobs?

Reply:
This time out is used only when the program exits, in fact this program will run until EBS backups in every k8s cluster finished. (At that context, the original context has been canceled.)

Reply:
if so, we don't need a timeout here, right?

Reply:
I just wondering how other operators(if exists) use this command